src/test/unit/function-info.test.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { stripFunctionArguments } from '../../profile-logic/function-info';
+
+describe('get-function-name', function () {
+  it('should strip the function arguments', function () {
+    expect(stripFunctionArguments('ns::fn()')).toEqual('ns::fn');
+    expect(stripFunctionArguments('ns::fn(bool (*)(JS::Handle<JSObject*>))')).toEqual('ns::fn');
+    expect(stripFunctionArguments('Bar* ns::fn<(Foo)0>(void const*, unsigned int)')).toEqual('Bar* ns::fn<(Foo)0>');
```

Comment:
I wonder, is this what one would expect, or should the template part also be removed?